src/Views/AppInfoView.vala

```diff
@@ -219,13 +212,11 @@ namespace AppCenter.Views {
             add (scrolled);
 
             open_button.get_style_context ().add_class ("h3");
-            reload_css ();
-        }
 
-        protected override void set_up_package (uint icon_size = 48) {
-            package_summary.label = package.get_summary ();
-            package_summary.ellipsize = Pango.EllipsizeMode.END;
-            base.set_up_package (icon_size);
+            base.set_up_package (128);
```

Comment:
I believe `base.` isn't needed here since there is not overriding function in the current class anymore.